test/action.yml

```diff
@@ -72,21 +71,21 @@ runs:
     # Print error logs from snap(s)
     - if: failure()
       shell: bash
-      working-directory: ${{github.action_path}}
+      working-directory: ${{github.workspace}}/src/test
       run: ./print-errors.sh ${{inputs.name}}
 
-    # For some reason, github.action_path resolves to a wrong path when directly passed to
+    # For some reason, github.workspace resolves to a wrong path when directly passed to
     # upload-artifact's path or in bash
     - if: always()
       shell: bash
-      working-directory: ${{github.action_path}}
+      working-directory: ${{github.workspace}}/src/test
       run: |
-        echo "action_path=$PWD" >> $GITHUB_ENV
+        echo "workspace=$PWD" >> $GITHUB_ENV
         
     # Upload snap logs
     - if: always() 
       uses: actions/upload-artifact@v3
       with:
         name: ${{inputs.name}}-logs
-        path: ${{env.action_path}}/suites/${{inputs.name}}/*.log
+        path: ${{env.workspace}}/suites/${{inputs.name}}/*.log
         retention-days: 30
```

Comment:
The workaround is no longer needed because that is only needed for action variables when being passed to another action.

It should be possible to use `${{github.workspace}}/src/test` directly there needed.

Reply:
Updated